src/MICore/LaunchOptions.xsd

```diff
@@ -300,6 +290,11 @@
               <xs:documentation>Any arguments to pass to this program.</xs:documentation>
             </xs:annotation>
           </xs:attribute>
+          <xs:attribute name="PipeCommandArguments" type="xs:string" use="optional">
```

Comment:
I had trouble understanding what this is for from the name+description.

How about:
Name: PipeUnixShellArguments
Description: Arguments passed to the pipe executable to open up a remote Unix (ex: bash or similar) shell. This is used for running 'kill' on the remote system, or other commands that the debugger may need. If not specified, the debugger will do its best without the shell. But some features, such as setting breakpoints in run mode, may not work.